docs/tools-resources/faq.mdx

```diff
@@ -0,0 +1,112 @@
+---
+description: Conway Ledger Era FAQ"
+sidebar_position: 2
+---
+
+# Conway Ledger Era FAQ
+
+**Warning:** due to on-going development some answers may change over time.
+
+*Last Update: 2023-09-18*
+
+Based on Ledger Design at commit: [cd10d6b](https://github.com/input-output-hk/cardano-ledger/blob/cd10d6b968bda704fb1ddc1226388fe221789155/eras/conway/test-suite/cddl-files/conway.cddl
+)
+
+### 1. What are the new Conway certificates?
+
+| Certificate in CDDL            | Description                                                              |
+|------------------------------- |------------------------------------------------------------------------- |
+| `reg_cert`                     | Stake credential registration                                            |
+| `unreg_cert`                   | Stake credential un-registration                                         |
+| `vote_deleg_cert`              | Vote delegation                                                          |
+| `stake_vote_deleg_cert`        | Delegates to a stake pool and a DRep at once                             |
+| `stake_reg_deleg_cert`         | Registers stake credential and delegates to a stake pool                 |
+| `vote_reg_deleg_cert`          | Registers stake credential and delegates to a DRep                       |
+| `stake_vote_reg_deleg_cert`    | Registers stake credential, delegates to a pool, and delegates to a DRep |
+| `auth_committee_hot_cert`      | Authorize Constitutional Committee hot credential                        |
+| `resign_committee_cold_cert`   | Resign Constitutional Committee cold credential                          |
+| `reg_drep_cert`                | Register DRep Credential                                                 |
+| `unreg_drep_cert`              | Unregister (retire) DRep Credential                                      |
+| `update_drep_cert`             | Update DRep metadata anchor                                              |
+
+### 2. Why are there now two ways to register/ unregister stake credentials?
+
+⚠ Because the old `stake_registration` and `stake_deregistration` are to be depreciated in the era following Conway ⚠ 
+
+The new version allows for leger to check deposit amounts and return instantly, matching the other new deposit containing certificates.
+
+### 3. Why do new certificates have a `coin` field?
+
+This has been done so deposit amounts can be checked on the Ledger level. This is also more useful under Voltaire since protocol params are likely to change more frequently than past eras.
```

Comment:
```suggestion
This has been done so deposit amounts can be checked on the Ledger level. This is also more useful under Voltaire since protocol parameters are likely to change more frequently than past eras.
```